### ISSUE TITLE
Pro 5 running and scrolling verry slow on startup

### ISSUE THREAD

**Issue by ghost** [ubports/anbox#16]
Device: PRO 5
Channel: 16.04/rc
Anbox Version: 2018-09-16 whatever is most recent.

Hi Marius, 
i have found out something when i startup my Pro 5 into the morning it seems something is doing things in the background and eat a lot of memory of the system, as scrolling go verry slow and not smooth like normoly i dont know if that would be a bug or not.

I have found out when you let be the Pro 5 for 10 mins then it runs and scroll again like it has to be.

BTW: I have the same Android apps also on my FP2 and on the FP2 i dont have this issue. About getting hot on FP2 i dont have that also not.

**Comment by Stefano0101** [comment]
yes, it happened on mine PRO 5 as well. Like it froze for a while, 15-20 seconds, then nothing works, not even possible to reboot or power off from the system menu. Restart the phone was the only option. It did not crash the Unity, though.

**Comment by ghost** [comment]
Also today it froze compleed my Pro 5 a few times.

**Comment by Stefano0101** [comment]
It's happening even when Anbox is not running, when is disabled.

**Comment by ghost** [comment]
Then it seems a problem with the kernel or anbox image i think... i marked as a bug.

It seems on my FP2 i dont have this problem..

**Comment by tomoqv** [comment]
This is happening more and more frequently on my Pro 5. Last 24 hours, I have rebooted 4 times because of this. I will disable Anbox again and see if it helps.
